<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乙方可视化编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;宋体&quot;;
font: 75 10pt &quot;Arial&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menu_Layer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>图层(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_DeleLayer"/>
    <addaction name="action_showAttr"/>
    <addaction name="action_Con"/>
   </widget>
   <widget class="QMenu" name="menu_Point">
    <property name="title">
     <string>点编辑(&amp;P)</string>
    </property>
    <addaction name="action_AddPoint"/>
    <addaction name="action_DelePoint"/>
    <addaction name="action_MovePoint"/>
    <addaction name="CopyPoint"/>
    <addaction name="action_ModPoint"/>
   </widget>
   <widget class="QMenu" name="menu_Line">
    <property name="title">
     <string>线编辑(&amp;L)</string>
    </property>
    <addaction name="action_AddLine"/>
    <addaction name="action_DeleLine"/>
    <addaction name="action_MoveLine"/>
    <addaction name="CopyLine"/>
    <addaction name="action_PointFromLine"/>
    <addaction name="EditType"/>
    <addaction name="PntDelFromLine"/>
    <addaction name="action_ModLine"/>
   </widget>
   <widget class="QMenu" name="menu_Poly">
    <property name="title">
     <string>面编辑(R)</string>
    </property>
    <addaction name="action_AddPoly"/>
    <addaction name="action_DelePoly"/>
    <addaction name="action_MovePoly"/>
    <addaction name="action_ModPoly"/>
    <addaction name="CopyPolygon"/>
    <addaction name="Edit_Polygon"/>
    <addaction name="PntAddToLine_Polygon"/>
    <addaction name="PntDelFromLine_Polygon"/>
   </widget>
   <widget class="QMenu" name="menu_General">
    <property name="title">
     <string>通用编辑(&amp;G)</string>
    </property>
    <addaction name="action_Delete"/>
    <addaction name="Select"/>
   </widget>
   <widget class="QMenu" name="menu_Basic">
    <property name="title">
     <string>基本操作(&amp;B)</string>
    </property>
    <addaction name="action_Enlarge"/>
    <addaction name="action_Narrow"/>
    <addaction name="action_Resize"/>
    <addaction name="action_StartEdit"/>
    <addaction name="action_SaveEdit"/>
    <addaction name="action_StopEdit"/>
    <addaction name="action_Property"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_Layers"/>
    <addaction name="action_properties"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>空间分析</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Basic"/>
   <addaction name="menu_Point"/>
   <addaction name="menu_Line"/>
   <addaction name="menu_Poly"/>
   <addaction name="menu_General"/>
   <addaction name="menu_View"/>
   <addaction name="menu"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Resize"/>
   <addaction name="action_Enlarge"/>
   <addaction name="action_Narrow"/>
   <addaction name="action_StartEdit"/>
   <addaction name="action_Property"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Layer">
   <property name="minimumSize">
    <size>
     <width>109</width>
     <height>162</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图层</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="listLayer">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Param">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>140</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图形参数设置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_LineType">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>实线</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>虚线</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>画笔线型：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QSpinBox" name="spinBox_LineWeight">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>画笔宽度：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>141</width>
       <height>110</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QTextBrowser" name="textBrowser_pen">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="color_pen">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>画笔颜色：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>141</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>填充颜色：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_fill">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_tianchong">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>颜色库</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_AddPoint"/>
   <addaction name="action_DelePoint"/>
   <addaction name="action_AddLine"/>
   <addaction name="action_DeleLine"/>
   <addaction name="CopyLine"/>
   <addaction name="action_PointFromLine"/>
   <addaction name="action_4"/>
   <addaction name="action_AddPoly"/>
   <addaction name="action_ModPoly"/>
   <addaction name="action_showAttr"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_NEW_DOCUMENT.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>新建图层(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>../picture/IDB_OPEN_DOCUMENT.bmp</normaloff>
     <normalon>../picture/IDB_OPEN_DOCUMENT.bmp</normalon>../picture/IDB_OPEN_DOCUMENT.bmp</iconset>
   </property>
   <property name="text">
    <string>打开图层(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save As">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_Con">
   <property name="icon">
    <iconset>
     <normalon>../picture/lianjie.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_AddPoint">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_EDIT_CTRLPNT.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>添加点</string>
   </property>
  </action>
  <action name="action_MovePoint">
   <property name="text">
    <string>移动点</string>
   </property>
  </action>
  <action name="action_DelePoint">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EDIT_PNT_DEL.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>删除点</string>
   </property>
  </action>
  <action name="action_ModPoint">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EDIT_PNT_GRAPHE.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>修改点参数</string>
   </property>
  </action>
  <action name="action_AddLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EditTool_InputLine.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>添加线</string>
   </property>
  </action>
  <action name="action_DeleLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_BUTTON_EDIT_DELETE.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>删除线</string>
   </property>
  </action>
  <action name="CopyLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_BUTTON_CUTLIN.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>复制线</string>
   </property>
   <property name="toolTip">
    <string>复制线</string>
   </property>
  </action>
  <action name="action_LjLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_SELECT_LIN.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>连接线 </string>
   </property>
  </action>
  <action name="action_ModLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EditTool_AttributeModify修改参数.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>修改线参数</string>
   </property>
  </action>
  <action name="action_PointFromLine">
   <property name="icon">
    <iconset>
     <normalon>../picture/NewShape.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>线上加点</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_BUTTON_DELETEPNT.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>线上删点</string>
   </property>
  </action>
  <action name="action_AddPoly">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EditTool_InputPoint.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>添加面 </string>
   </property>
  </action>
  <action name="action_MovePoly">
   <property name="text">
    <string>移动面</string>
   </property>
  </action>
  <action name="action_ModPoly">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EditTool_InputRegion.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>修改面参数</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset>
     <normalon>../picture/clear.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>拉框选择</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>圆选择</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>多边形选择</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>按属性选择</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>删除图元</string>
   </property>
  </action>
  <action name="action_Enlarge">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_ZOOM_IN.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_Narrow">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_ZOOM_OUT.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_Resize">
   <property name="icon">
    <iconset>
     <normalon>../picture/fullExtent.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </action>
  <action name="action_StartEdit">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_EditTool_ExtractFeature.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>开始编辑</string>
   </property>
  </action>
  <action name="action_Property">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_SET_CTRLPNT_PARAM.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>图形参数设置</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>删除线</string>
   </property>
  </action>
  <action name="action_DelePoly">
   <property name="text">
    <string>删除面</string>
   </property>
  </action>
  <action name="action_MoveLine">
   <property name="text">
    <string>移动线</string>
   </property>
  </action>
  <action name="action_StopEdit">
   <property name="icon">
    <iconset>
     <normalon>../picture/delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>结束编辑</string>
   </property>
  </action>
  <action name="action_SaveEdit">
   <property name="icon">
    <iconset>
     <normalon>../picture/IDB_SAVE_DOCUMENT.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>保存编辑</string>
   </property>
  </action>
  <action name="Select">
   <property name="text">
    <string>选择图元</string>
   </property>
  </action>
  <action name="CopyPoint">
   <property name="text">
    <string>复制点</string>
   </property>
  </action>
  <action name="EditType">
   <property name="text">
    <string>线上移点</string>
   </property>
  </action>
  <action name="PntDelFromLine">
   <property name="text">
    <string>线上删点</string>
   </property>
  </action>
  <action name="CopyPolygon">
   <property name="text">
    <string>复制面</string>
   </property>
  </action>
  <action name="Edit_Polygon">
   <property name="text">
    <string>线上移点</string>
   </property>
  </action>
  <action name="PntAddToLine_Polygon">
   <property name="text">
    <string>线上加点</string>
   </property>
  </action>
  <action name="PntDelFromLine_Polygon">
   <property name="text">
    <string>线上删点</string>
   </property>
  </action>
  <action name="action_DeleLayer">
   <property name="text">
    <string>删除图层</string>
   </property>
  </action>
  <action name="action_showAttr">
   <property name="icon">
    <iconset>
     <normalon>../picture/ID_BUTTON_MODIFYFEATATR.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>查看属性</string>
   </property>
  </action>
  <action name="action_Layers">
   <property name="text">
    <string>图层</string>
   </property>
  </action>
  <action name="action_properties">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>图元求交</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>图元求并</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>图元求差</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
